### PR TITLE
edgeAgent asp metrics

### PR DESCRIPTION
**Pull request by lfitchett**

### CODE REVIEW

File: edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs [hunk 1]
```diff
@@ -114,6 +114,8 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                     storageBackupPath = Option.Some(GetOrCreateDirectoryPath(configuration.GetValue<string>("BackupFolder"), EdgeAgentStorageBackupFolder));
                 }
 
+                backupConfigFilePath = GetFullBackupFilePath(storagePath, backupConfigFilePath);
```
Comment (CindyXing) [inline edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs:117]: What's the reason for this change to be in this PR?

**Comment by lfitchett** [comment]
Opened pr on dave's branch, will merge with his and he will open pr in this repo with combined changes.
